src/patternfly/components/Nav/nav.scss

```diff
@@ -218,49 +133,28 @@
     position: relative;
     display: flex;
     align-items: baseline;
-    padding:
-      var(--pf-c-nav__list-link--PaddingTop)
-      var(--pf-c-nav__list-link--PaddingRight)
-      var(--pf-c-nav__list-link--PaddingBottom)
-      var(--pf-c-nav__list-link--PaddingLeft);
+    padding: var(--pf-c-nav__list-link--PaddingTop) var(--pf-c-nav__list-link--PaddingX) var(--pf-c-nav__list-link--PaddingBottom);
     font-weight: var(--pf-c-nav__list-link--FontWeight);
     color: var(--pf-c-nav__list-link--Color);
+    background-color: var(--pf-c-nav__list-link--BackgroundColor);
 
     @media screen and (min-width: $pf-global--breakpoint--md) {
-      --pf-c-nav__list-link--PaddingRight: var(--pf-c-nav__list-link--md--PaddingRight);
-      --pf-c-nav__list-link--PaddingLeft: var(--pf-c-nav__list-link--md--PaddingLeft);
+      --pf-c-nav__list-link--PaddingX: var(--pf-c-nav__list-link--md--PaddingX);
```

Comment:
Should this change be grouped with the [other variables](#diff-6416b4627393c3e840a14c791aa6d896L155)?

Reply:
should I leave this here after our discussion with the css group

Reply:
If I understood from the conversation Wednesday, you would want this media query right after where you define `--pf-c-nav__list-link--PaddingX` in the top of the CSS file.

Reply:
Looks like the same would apply to all of the media queries in this file.